test/fixtures/dialects/snowflake/snowflake_copy_into_location.sql

```diff
@@ -0,0 +1,7 @@
+COPY INTO '@public.cb_data_archive_prod/airflow-pipelines/2022-11-01/model/fct_rolling_active_users_l28/'
```

Comment:
That's quite a lot of syntax added yet only one more test case. Does that cover it all (I've not looked closely btw).

There's quite a lot more examples at:
https://docs.snowflake.com/en/sql-reference/sql/copy-into-location.html#examples
and
https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html#examples

Are any of them good test cases to add?

Reply:
This is a very good point! I will try to parse the other two format to see if sqlfluff could parse it successfully